--- deploy.prototxt.orig	2017-02-05 15:41:48.590564704 +0000
+++ deploy.prototxt	2017-02-05 15:42:42.368842998 +0000
@@ -2170,15 +2170,3 @@
     }
   }
 }
-layer {
-  name: "cluster"
-  type: "Python"
-  bottom: "coverage"
-  bottom: "bboxes"
-  top: "bbox-list"
-  python_param {
-    module: "caffe.layers.detectnet.clustering"
-    layer: "ClusterDetections"
-    param_str: "1024, 1024, 16, 0.6, 3, 0.02, 22, 1"
-  }
-}
